server/protocols/jmap/src/test/java/org/apache/james/jmap/model/mailbox/MailboxUpdateRequestTest.java

```diff
@@ -0,0 +1,60 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.jmap.model.mailbox;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Optional;
+
+import org.junit.Test;
+
+public class MailboxUpdateRequestTest {
+    
+    @Test
+    public void getParentIdShouldReturnEmptyWhenNotGiven() throws Exception {
+        //Given
+        MailboxUpdateRequest testee = MailboxUpdateRequest.builder().name("my box").build() ;
+        //When
+        Optional<String> actual = testee.getParentId();
+        //Then
+        assertThat(actual).isEmpty();
+    }
+
+    @Test
+    public void getParentIdShouldReturnNullWhenNullParentIdGiven() throws Exception {
+        //Given
```

Comment:
Same comment:
We have to check different kind of input values for parentId:
- the parentId is not given -> Optional.empty
- the parentId is null (set this mailbox as a root) -> null
- the parentId is given -> Optional.of